code/game/objects/items/stacks/nanopaste.dm

```diff
@@ -37,20 +37,25 @@
 
 		if(S)
 			if(BP_IS_ROBOTIC(S))
+				if(!S.get_damage())
+					to_chat(user, SPAN_NOTICE("There is nothing to fix here."))
+					return
+				for(var/datum/wound/W in S.wounds)
+					if(W.internal)
+						return
+					if(amount <= 0)
+						break
+					if(!do_mob(user, M, W.damage/5) && use(1))
+						to_chat(user, SPAN_NOTICE("You must stand still to repair \the [S]."))
+						break
+					if(!can_use(1))
```

Comment:
Wait wait wait. It will use(1) at line 48 but check for can_use(1) at 51. May be they should be swapped.

Reply:
No, line 48 is when the item would have a stack of itself be actually used.

use() does not inform the user when the item is used up. This is for player feedback purposes.
I did notice that how this is formatted currently would prevent people from doing surgery on damaged robotic limbs with the nanopaste. Push incoming.